test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerTest.cs

```diff
@@ -153,6 +177,139 @@ public void Controller_View_WithParameterViewNameAndViewModel_SetsResultViewName
         }
 
         [Fact]
+        public void Controller_View_WithNullModelParameter_MaintainsNullModel()
```

Comment:
Watch naming. This case is about _overriding_ the `Model` and setting it to `null`.